contracts/gateway/EIP20CoGateway.sol

```diff
@@ -202,23 +203,23 @@ contract EIP20CoGateway is GatewayBase {
      * @param _utilityToken The utility token address that will be used for
      *                      minting the utility token.
      * @param _core Core contract address.
-     * @param _bounty The amount that facilitator will stake to initiate the
+     * @param _bounty The amount that facilitator will stakes to initiate the
```

Comment:
😳 
`The amount that facilitator stakes to initiate the stake process`

Reply:
👍